prepare_stmt.go

```diff
@@ -32,6 +32,15 @@ func (db *PreparedStmtDB) GetDBConn() (*sql.DB, error) {
 	return nil, ErrInvalidDB
 }
 
+func (db *PreparedStmtDB) Ping() error {
+	sqldb, err := db.GetDBConn()
```

Comment:
`GetDBConn` will return error if the underlying database connection is not a *sql.DB or does not implement `GetDBConnector`. We need to avoid this error because it is not a ping error but the driver does not implement the ping interface.

Reply:
You need to check this error otherwise if we can't get the connection we can't ping. Therefore it is a ping error. If we don't check this error open will return a DB and a nil error.

Reply:
@jinzhu What are your thoughts on this PR?